Sources/SymbolKit/UnifiedSymbolGraph/GraphCollector.swift

```diff
@@ -126,7 +131,11 @@ extension GraphCollector {
     ///   - url: The file name where the symbol graph is located.
     /// - Returns: The name of the module described by `graph`, and whether the symbol graph is a "primary" symbol graph.
     public static func moduleNameFor(_ graph: SymbolGraph, at url: URL) -> (String, Bool) {
-        let isMainSymbolGraph = !url.lastPathComponent.contains("@")
+        // The Swift compiler generates symbol graph URLs that contain an `@` symbol to denote an
+        // extension to another modules.
+        // The snippet extractor generates symbol graph files without an `@` symbol and with the
+        // the metadata that indicates the module `isVirtual`.
```

Comment:
Thanks for explaining this in a code comment.

Reply:
You're very welcome!

Lol - it took me two days to hunt down that detail, It was GETTING documented ;-)